presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -479,6 +479,30 @@ Alter table operations are supported in the connector::
 
      ALTER TABLE iceberg.web.page_views DROP COLUMN location;
 
+TRUNCATE
+^^^^^^^^
+
+The iceberg connector can delete all of the data data from tables without
+dropping the table from the metadata catalog using ``TRUNCATE TABLE``.
+
+.. code-block:: sql
+
+    TRUNCATE TABLE nation;
+
+.. code-block:: text
```

Comment:
```suggestion
.. code-block:: sql
```

Reply:
This is actually the output from the CLI after running `TRUNCATE TABLE nation`, so I think keeping `text` here is ok.

Reply:
Ah, got it!